data/mods/CRT_EXPANSION/items/crt_tools.json

```
@@ -66,7 +66,13 @@
     "description": "CRIT melee weapon.  Alien runes adorn the carbon steel blade.  The blade oddly seems to lack sharpness, and yet upon closer oberservation, a hum of energy thrums from within.",
     "color": "dark_gray",
     "material": [ "hardsteel" ],
-    "artifact_data": { "charge_type": "ARTC_NULL", "effects_wielded": [ "AEP_DEX_UP", "AEP_SPEED_UP", "AEP_SPEED_UP" ] },
+    "//": "Legacy artifact data is set for always on wield dex +2 and speed up times 2",
+    "relic_data": {
+      "passive_effects": [
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 20 } ] },
```

```suggestion
        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "add": 40 } ] },
```
the legacy AEP_SPEED_UP is +20